### PR TITLE
Updated 404 page

### PR DESCRIPTION
**Pull request by Lukewh**
# Done

- Removed logo from the 404 page

# QA

- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/asdlkgjwjnbnrjnb

OR

- Visit http://snapcraft.io-pr-374.run.demo.haus/asdlkgjwjnbnrjnb

# Screenshot

screenshot from 2018-03-13 13-25-27

### CODE REVIEW

File: templates/404.html [hunk 1]
```diff
@@ -4,9 +4,9 @@
 
 {% block content %}
 <div class="p-strip">
-  <div class="row">
-    <div class="col-6">
-      <img src="/static/confused-hex.svg" />
```
Comment (bartaz) [inline templates/404.html]: We should remove the `confused-hex.svg` file from project if it's not used anymore.

**Comment by lyubomir-popov** [comment]
@Lukewh Just consulted with Tom, we shouldn't use the bird for things like that, although it looks nice (sorry!). Having a creative 404 page is a good idea, but we don't have visuals that work with the message at the moment. So can we remove it and have the copy flush left?

**Comment by Lukewh** [comment]
@lyubomir-popov sounds good, was just an idea Greg had on the sprint (as the current 404 isn't "on brand" in any way). I'll update this PR and ping you

**Comment by lyubomir-popov** [comment]
@Lukewh Looks good. One minor niggle, more of a ux thng so maybe something for @Greggless - at the moment there's a bit of repetition - 
"Page not found"
"Snap not found: {snap-name}"

Could these be combined into one? would help if the snap name doesn't look like fine print.

Also shouldn't we be providing a helpful suggestion - maybe a search input?

**Comment by lyubomir-popov** [comment]
(Adding design plus one as the above comments are not a blocker from visual point of view.)

**Comment by Greggless** [comment]
We should definitely design a more useful 404. 

This'll do (I hate saying that) for now, but I'll open an issue for something more useful

**Pull request merged**